src/msg/async/AsyncConnection.cc

```diff
@@ -432,9 +429,7 @@ void AsyncConnection::process()
 
       case STATE_OPEN_MESSAGE_HEADER:
         {
-#if defined(WITH_LTTNG) && defined(WITH_EVENTTRACE)
-          ltt_recv_stamp = ceph_clock_now();
-#endif
+          recv_stamp = ceph_clock_now();
```

Comment:
this changes how we updates `recv_stamp`. hence not in the scope of "cleanup"? moreover, there is no need to update `recv_stamp` at this moment.